sources/AppBundle/Controller/Planete/ArticlesController.php

```diff
@@ -21,13 +21,15 @@ public function __construct(
 
     public function __invoke(Request $request): Response
     {
+        $prePage = 20;
```

Comment:
```suggestion
        $perPage = 20;
```

Juste une petite typo de rien du tout

Reply:
Ah bien vu merci 😊